src/main/java/christmas/domain/Day.java

```diff
@@ -0,0 +1,48 @@
+package christmas.domain;
+
+import static christmas.domain.constants.DayConstants.*;
+import static christmas.exception.ExceptionConstants.DATE;
+
+import christmas.exception.InputException;
+
+public class Day {
+
+    private final int day;
+
+    public Day(final int day) {
+        validateDay(day);
+        this.day = day;
+    }
+
+    private void validateDay(final int day) {
+        if (day > LAST.getNumber() || day < FIRST.getNumber()) {
+            throw InputException.createException(DATE);
+        }
+    }
+
+    public boolean isChristmas() {
+        return day <= CHRISTMAS.getNumber();
+    }
+
+    public int getChristmasDiscount() {
+        return THOUSAND.getNumber() + (day - FIRST.getNumber()) * HUNDRED.getNumber();
+    }
+
+    private int modDay() {
+        return day % WEEK.getNumber();
+    }
+
+    public boolean isWeekend() {
+        for (int i = FIRST.getNumber(); i < START_WEEKDAY.getNumber(); i++) {
+            if (modDay() == i) {
+                return true;
+            }
+        }
+        return false;
```

Comment:
(이런 방법도 있어요!)
이렇게 사용하면 조금 더 간결하게 표현할 수 있습니다 :)
```suggestion
           return IntStream.range(FIRST.getNumber(), START_WEEKDAY.getNumber())
                .anyMatch(i -> modDay() == i);
```